### ISSUE TITLE
Report system

### ISSUE THREAD

**Issue by Nallefjant** [ValveSoftware/Dota2-Gameplay#15938]
### Description

So I just got into low prio, because I played an unranked game. I am Immortal, my allies were Divine and unranked. They kept dying 1 by 1 out of position because of their lack of map awareness, so I were farming until everyone spawned so that we could actually take a proper fight before they were taking high ground. But since they died and saw that I were just farming instead of pretending to try to help them (it was too late, they were too far away) they thought I were griefing and reported me. And when they were taking high ground I started defending and they started being afk so I reported them back and after the game I got into low prio and I woke up this morning and saw that all 4 of them got "ACTION TAKEN" against them as well.

How does it make sense that I get into low prio for farming while they feed? I weren't originally going to report them for playing bad, after all it was unranked - it is to be expected. Does it make any sense that everyone in a team gets punished or is it in fact just a shit match and nobody should punished?

Also, when getting reported you should be able write something in your defense. How does it make sense that I should play suboptimal - stop farming, teleport back and pretend that it is a good idea to take a fight 4v5 after one of us just died? Just to avoid getting reports?

**Comment by chester-simps** [comment]
I completely agree that this patrol system is complete bullshit! I played with an anti-mage at position 3 and a furion at position 5, who spent 0 minutes on the lane with the carry and even in the dotabuff his position is pointed as the roamer, and even though he lost while roaming(dotabuff opinion). As a result, I lost a lane with two decent enemies. And after this match I was banned, great! I'm just looking at the "Incoming match player report" on Steam and it says I only received reports for one match, guess which match, and it was 5 reports, NICE. And after all of that i reported these grifiers, and guess what??? Nothing happened, these dogs continue to play without penalties. GOOD, VOLVO, GOOD!

proof match ID 7656810042
